flake.nix

```diff
@@ -460,6 +460,9 @@
                 nativeBuildInputs = shellCommonNative.nativeBuildInputs ++ [ pkgs.yarn pkgs.nodejs ];
                 shellHook = ''
                   export FEDIMINT_UI_SHELL=1
+                  cd fedimint-ui
```

Comment:
assumes we executed `nix .#fedimint-ui` from repo root, but that is not always the case... -\\-/-